modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx

```diff
@@ -78,6 +78,112 @@ describe('View Channel', () => {
 		expect(queryByLabelText('Edit')).toBeNull();
 	});
 
+	it('should check if DELETE and CLEAR DATA buttons are displayed', () => {
+		API.user.fetchCurrentUser.mockReturnValueOnce(
+			Promise.resolve(data.mockUser())
+		);
+
+		const {queryByText} = render(<DefaultComponent />);
+
+		jest.runAllTimers();
+
+		expect(queryByText('Delete')).toBeInTheDocument();
+		expect(queryByText('Clear Data')).toBeInTheDocument();
+	});
+
+	it('should check error modal message and hyperlink on deleting property that has CHANNELS synced', async () => {
+		API.user.fetchCurrentUser.mockReturnValueOnce(
+			Promise.resolve(data.mockUser())
+		);
+
+		API.channels.fetch.mockReturnValueOnce(
+			Promise.resolve(
+				data.mockChannel(1, 1, {
+					commerceChannelsCount: 5,
+					groupsCount: 0
+				})
+			)
+		);
+
+		const {container, getByText} = render(<DefaultComponent />);
+
+		jest.runAllTimers();
+
+		expect(
+			getByText(
+				'There are 0 Sites and 5 Channels synced to this property.'
+			)
+		).toBeInTheDocument();
+
+		const deleteBtn = container.querySelector(
+			'button[data-testid="delete"]'
+		);
+
+		fireEvent.click(deleteBtn);
```

Comment:
It is missing to check if the modal was opened. Create an expect here.

Reply:
Done!